pages/recipe/[id]/index.js

```diff
@@ -65,12 +76,29 @@ export default function DetailPage({
     }
   };
 
-  if (error || dataError) {
-    return <h1>Fehler...</h1>;
-  }
+  async function handleCollection(event) {
+    event.preventDefault();
+    const isDuplicate = user.collections.some((col) =>
+      col.recipes.map((recipe) => recipe.id === id)
+    );
+    if (isDuplicate) {
+      notifyError("Dieses Rezept ist bereits gespeichert.");
+      return;
+    }
```

Comment:
this logic means, a recipe can only be saved to one collection per user. Is this the desired behaviour?
Haha, I doubt it!